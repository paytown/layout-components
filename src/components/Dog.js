import React from 'react';
import dogPic from '../assets/timmothy.jpeg';

export default function Header() {
  const dog = {
    name: 'Jimmothy',
    age: '311',
    sign: 'virgo'
  };

  return (
    <main>
      <h3>Name</h3>
      <p>{dog.name}</p>

      <h3>Age</h3>
      <p>{dog.age}</p>

      <h3>Sign</h3>
      <p>{dog.sign}</p>

      <img src={dogPic} />
    </main>
  );
}
