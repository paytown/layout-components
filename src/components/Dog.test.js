import React from 'react';
import { shallow } from 'enzyme';
import Dog from './Dog';

describe('Dog Component', () => {
  it('show that pup', () => {
    const wrapper = shallow(<Dog />);
    expect(wrapper).toMatchSnapshot();
  });
});
