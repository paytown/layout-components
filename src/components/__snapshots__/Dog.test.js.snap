// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dog Component show that pup 1`] = `
<main>
  <h3>
    Name
  </h3>
  <p>
    Jimmothy
  </p>
  <h3>
    Age
  </h3>
  <p>
    311
  </p>
  <h3>
    Sign
  </h3>
  <p>
    virgo
  </p>
  <img
    src="/my/image"
  />
</main>
`;
